### PR TITLE
Incorporate digital objects into clipboard export guides

### PR DESCRIPTION
**Pull request by ablwr**
Hi @evelynPM, please compare this to your Issue #170 and let me know if it looks good and can be incorporated into the codebase!

Adding @fiver-watson for transparency and double peer review.

P.S.: the documentation I grabbed from your Google Doc -- the image for the first and second sets ARE the same, right? I used the same image for both sections.

### CODE REVIEW

File: user-manual/import-export/csv-export.rst
Comment (fiver-watson) [inline user-manual/import-export/csv-export.rst:368]: Everything looks good overall, but I wonder if we want to include a further clarification (in all the places where this note appears)? Something like: 

"This means you will need to individually add the descriptions with digital objects to the clipboard if you want to export them, rather than just adding a parent description."

Not a hard requirement, up to you!

File: user-manual/import-export/csv-export.rst
Comment (fiver-watson) [inline user-manual/import-export/csv-export.rst]: It would probably be good to add a link in here somewhere to the settings page, so administrators just learning about this functionality know where to go to configure it.

File: user-manual/import-export/export-xml.rst
Comment (fiver-watson) [inline user-manual/import-export/export-xml.rst]: Same comments - maybe a link to the settings page; maybe a clarification on the tradeoff of exporting digital objects vs hierarchies?

File: user-manual/administer/settings.rst
Comment (fiver-watson) [inline user-manual/administer/settings.rst:1107]: Is there anything we need to say about limitations, required available server storage space, etc?

**Review by fiver-watson — COMMENTED** [review]
Looks good - I've added a couple questions and suggestions but they are minor and up to your discretion :)

**Comment by ablwr** [comment]
Thank you so much @fiver-watson! I'll leave these up to @evelynPM, as she wrote the prose. Happy to make any changes on her behalf.

**Comment by ablwr** [comment]
:eyes: :question:

File: user-manual/administer/settings.rst
Comment (evelynPM) [reply user-manual/administer/settings.rst:1107]: At the end of the first paragraph, add "Administrators should ensure that the server on which the AtoM instance is installed has enough processing power, memory and storage space to create and temporarily store zipped packages of the digital objects selected for download."

File: user-manual/import-export/csv-export.rst
Comment (evelynPM) [reply user-manual/import-export/csv-export.rst]: In line 349, make "configured" a link to https://www.accesstomemory.org/en/docs/2.7/user-manual/administer/settings/#clipboard-settings.

File: user-manual/import-export/csv-export.rst
Comment (evelynPM) [reply user-manual/import-export/csv-export.rst:368]: Agreed.

File: user-manual/import-export/export-xml.rst
Comment (evelynPM) [reply user-manual/import-export/export-xml.rst]: Agreed, make "configured" a link to https://www.accesstomemory.org/en/docs/2.7/user-manual/administer/settings/#clipboard-settings. And add to line 484 "This means you will need to individually add the descriptions with digital objects to the clipboard if you want to export them, rather than just adding a parent description."

**Review by evelynPM — COMMENTED** [review]
I've address all of Dan's comments. I'm not quite sure what you mean by "the image for the first and second sets ARE the same, right? I used the same image for both sections."

**Comment by ablwr** [comment]
Made the updates. Please re-review again, and if it's fine, I can finish the process in gitolite and merge these changes into 2.7.

**Review by fiver-watson — CHANGES_REQUESTED** [review]
Sorry, minor change request. 

The links added currently like so: 

```
Some AtoM sites are :ref:`clipboard-settings` configured to enable users to ...
```

Won't actually turn "configured" into a link. Instead, it will insert the anchor's display heading (i.e. Clipboard Settings) right into that sentence! If you want to add a ref link on a different words as the anchor, you can try it like so: 

```
Some AtoM sites are :ref:`configured <clipboard-settings>`  to enable users to ...
```

**Comment by ablwr** [comment]
Not minor at all! Thank you, I thought I was following the right procedure.

**Comment by ablwr** [comment]
@fiver-watson @evelynPM How's this looking?

File: user-manual/administer/settings.rst
Comment (evelynPM) [inline user-manual/administer/settings.rst]: Suggest moving the last sentence of this paragraph to the end of the previous paragraph

**Comment by ablwr** [comment]
Thank you @evelynPM. @fiver-watson if I can get one last review from you, I can merge this in gitolite and close this PR.

**Comment by ablwr** [comment]
ping @fiver-watson I can merge this upon your approval

**Review by fiver-watson — APPROVED** [review]
Oops! Sorry @ablwr I missed the last update somehow. This looks great, thanks! 👍

**Comment by ablwr** [comment]
All set, cherry-picked into 2.7, closing this PR

**Pull request closed without merging**
